I have double check with the step by step guide with the tutorial from read.cash

littlenewthings opened the pull request:
I think it is done correctly. Hopefully owner can approve my request to update my addition into the slp-explorer?

blockparty-sh commented:
Hi. Sorry but this still isn't correct. For example

http://img.mywofs.com/nft/junk2glamour/original/c2327a0754b59b62291fd6560bc481e721d49d38d0ece26ded93acd2d6009caa.png

Is a 404 error. The tokenids needs to be accessible there with same folder structure like described in guide.

It looks like you just need to add .png extension to them from looking here

https://img.mywofs.com/nft/junk2glamour/original/

Let me know once you do and I can add. Thanks

littlenewthings commented:
Yes. I have checked through and updated with .png extension. My apologies that it must have happened when I renamed the files to the token names in ftp. 
I can see the images with my web browser now.

littlenewthings commented:
Hi, I have resolved the png extension issue. Please accept the changes?
